code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -149,7 +149,7 @@
 	if(is_open_container())
 		to_chat(user, SPAN_WARNING("\The [src] hasn't been secured yet!"))
 		return
-	if(do_after(user, 1 SECOND, TRUE))
+	if(do_after(user, 1 SECOND))
```

Comment:
```suggestion
	if(do_after(user, 1 SECOND, src, DO_UNIQUE))
```
This will restore the intended behavior of the autoinjector, which is that it is a unique action only done on one hand.

Reply:
This wouldn't work, the progress bar would be put ontop of the object that is inside an hand slot, making it invisible